src/unix/notbsd/linux/other/mod.rs

```diff
@@ -32,8 +36,247 @@ s! {
         __unused4: *mut ::c_void,
         __unused5: *mut ::c_void,
     }
+
+    // (bits/ioctl-types.h)
```

Comment:
It's ok to omit all the comments here and there, these files aren't really meant for general development as they're just reflections of what's found on the system. Any documentation should be found in local man pages and whatnot generally